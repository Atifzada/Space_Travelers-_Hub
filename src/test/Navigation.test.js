import React from 'react';
import { render } from '@testing-library/react';
import { BrowserRouter } from 'react-router-dom';
import Navbarmenu from '../components/Navigation';

it('renders correctly', () => {
  const container = render(
    <BrowserRouter>
      <Navbarmenu />
    </BrowserRouter>,
  );

  expect(container).toMatchSnapshot();
});
