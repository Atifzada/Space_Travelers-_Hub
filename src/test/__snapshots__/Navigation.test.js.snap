// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <nav
        class="navbar navbar-expand navbar-light bg-light"
      >
        <div
          class="navContainer container"
        >
          <span
            class="navTitle navbar-brand"
          >
            <img
              alt="Logo"
              class="d-flex align-top"
              height="70"
              src="planet.png"
              width="70"
            />
            Space Travelers' Hub
          </span>
          <div
            class="navlinks navbar-nav"
            id="links"
          >
            <a
              aria-current="page"
              class="active"
              href="/"
              style="text-decoration: underline; padding-right: 1rem; font-size: 1.5rem;"
            >
              Rockets
            </a>
            <a
              class=""
              href="/missions"
              style="font-size: 1rem;"
            >
              Missions
            </a>
            <a
              class=""
              href="/profile"
              style="font-size: 1rem;"
            >
              My Profile
            </a>
          </div>
        </div>
      </nav>
      <hr />
    </div>
  </body>,
  "container": <div>
    <nav
      class="navbar navbar-expand navbar-light bg-light"
    >
      <div
        class="navContainer container"
      >
        <span
          class="navTitle navbar-brand"
        >
          <img
            alt="Logo"
            class="d-flex align-top"
            height="70"
            src="planet.png"
            width="70"
          />
          Space Travelers' Hub
        </span>
        <div
          class="navlinks navbar-nav"
          id="links"
        >
          <a
            aria-current="page"
            class="active"
            href="/"
            style="text-decoration: underline; padding-right: 1rem; font-size: 1.5rem;"
          >
            Rockets
          </a>
          <a
            class=""
            href="/missions"
            style="font-size: 1rem;"
          >
            Missions
          </a>
          <a
            class=""
            href="/profile"
            style="font-size: 1rem;"
          >
            My Profile
          </a>
        </div>
      </div>
    </nav>
    <hr />
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
